lib/bundles/inspec-compliance/cli.rb

```diff
@@ -237,16 +237,13 @@ def upload(path) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Percei
     desc 'version', 'displays the version of the Chef Compliance server'
     def version
       config = Compliance::Configuration.new
-      if config['server_type'] == 'automate'
-        puts 'Version not available when logged in with Automate.'
+      info = Compliance::API.version(config)
+      if !info.nil? && info['version']
```

Comment:
It might be nice to have `Compliance::API.version` return an empty hash instead of `nil` to avoid having to do this extra check.